tests/Tests/200__monitor_and_manage/220__grafana/220__test_verify_port_forwarding_grafana.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Documentation   Verify that MT-SRE can connect to Grafana-dashboard using port-forwarding
+...
+...             = Variables =
+...             | PROC             | Required |        To store process id|
+
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+Resource       ../../Resources/Page/OCPDashboard/Monitoring/Grafana.robot
+
+
+*** Variables ***
+${PROC}     none
+
+
+*** Test Cases ***
+Verify that MT-SRE can connect to Grafana using port-forwarding
+    [Documentation]  Verifies that Grafana is accessible by MT-SRE when using oc port-forwarding
+    [Tags]   Tier2
+    ...      ODS-754
+
+    # Port Forwarding
+    ${PROC} =  Enable Access To Grafana Using OpenShift Port Forwarding
+    # It takes aroung 7 sec for the website to come up
+    Sleep  7s
+    ${response} =   Run   curl -I "http://localhost:3001"   # robocop: disable
+    Run Keyword If    "HTTP/1.1 200 OK" not in $response    Fail
+    ...       ELSE     Log To Console    SRE can connect to Grafana using prot-forwarding"
+    Open Browser  http://localhost:3001  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Verify If Jupyterhub SLI Is Present
+    Get WebElements    //*[contains(@aria-label,"Jupyterhub SLIs")]
+    [Teardown]  Disable Access To Grafana Using OpenShift Port Forwarding  ${PROC}
+
+
+*** Keywords ***
+Disable Access To Grafana Using OpenShift Port Forwarding
```

Comment:
Same as we have in ODS.robot the port-forwarding startup, the same file can include `Disable Access To Grafana Using OpenShift Port Forwarding  $port_forwarding_process`  with the Terminate Process

The test can have a test teardown calling that method and Close Browser